charts/plgd-hub/templates/certificate-authority/config.yaml

```diff
@@ -45,6 +45,6 @@ data:
       keyFile: {{ .signer.keyFile | default ( printf "%s/%s" $.Values.certificateauthority.ca.volume.mountPath $.Values.certificateauthority.ca.key ) | quote }}
       validFrom: {{ .signer.validFrom | quote }}
       expiresIn: {{ .signer.expiresIn | quote }}
-      hubId: {{ required "certificateauthority.signer.hubId or coapgateway.hubId or global.hubId is required" ( .signer.hubId | default ( $.Values.coapgateway.hubId | default $.Values.global.hubId)) | quote }}
+      hubID: {{ required "certificateauthority.signer.hubId or coapgateway.hubId or global.hubId is required" ( .signer.hubId | default ( $.Values.coapgateway.hubId | default $.Values.global.hubId)) | quote }}
```

Comment:
certificateauthority.signer.**hubId**,  .signer.**hubId**

Reply:
if I understand what @ondrejtomcik correctly, it is correct that the helm variables are still written as `hubId`. @ondrejtomcik can you please verify?

Reply:
Yes. Changing it there would be a breaking change for the helm chart.